static/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
Had ik toegevoegd, maar kan weg. Robots.txt is bedoeld voor zoekmachines, maar is voor deze web app niet relevant.